### PR TITLE
feat(bem): Make the searchBox BEMish

### PR DESCRIPTION
**Pull request by pixelastic**
I've also updated the `poweredBy` as well. I've changed the logic of
adding inline `style` to using a specific class (defined in the
default `css` file). I've switched the display/hide logic. We no
longer need to define a `display=true` in the props to show it, but
a `hidden=true` to hide it instead.

BREAKING CHANGE: The `poweredBy` component now expect a `hidden=false`
to be hidden, instead of a `display=true` to be displayed.

### CODE REVIEW

**Comment by vvo** [comment]
No need to modify it but it's "feat(searchBox)" BEM is not a feature per se :D

**Comment by pixelastic** [comment]
I'll remove the display/hide logic from the poweredBy component completly in another PR.

**Comment by vvo** [comment]
> I'll remove the display/hide logic from the poweredBy component completly in another PR.

Yep good for me as it was already the case before, see #189

File: components/PoweredBy/powered-by.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.ais-powered-by {
+  display: block;
```
Comment (vvo) [inline components/PoweredBy/powered-by.css]: not sure we need `display:block`, it's the default right? It should work without it

**Comment by vvo** [comment]
Just need to rebase now, I think that's because we merged PR that were newer than yours (the famous "please merge in order" mantra).

That's why we need to get PR merged fast, like water flowing :pray:

**Comment by pixelastic** [comment]
Yep, my bad. This should be good now.

**Pull request merged**